nss-day-cohort-29/nutshell-hope-floats
# User Register

# Description

created login/register form! when the user presses the register button the data that they inputted will be stored in the database.json



## Type of change

Please delete options that are not relevant.



# Testing Instructions for Change Made

git fetch --all
git checkout mr-first-branch
src/lib and run grunt!
Test to see if the info that you input in the login form when register button is pressed is transferred into the database.json


# Checklist:
